### PR TITLE
Chore/file size increase

### PR DESCRIPTION
**Pull request by gweiying**
## Problem

We want to support file uploads of size 15-20MB. 

## Solution

This PR increases the supported file upload size to 20MB. Note that uploading large files takes longer as the file is sent to Cloudmersive for virus scanning. On average, 20MB files take 25s for uploading, but can also take up to 55s for uploading. If the response time goes beyond 1 min, the upload fails and the user is prompted to try again.

Others
- Fixed bug where bulk upload and file upload input was using the same file size limit
- ~Added util file to parse byte sizes~ Unneeded

### CODE REVIEW

**Review by halfwhole — APPROVED** [review]
LGTM

File: src/client/user/components/CreateUrlModal/CreateLinkForm.tsx [hunk 1]
```diff
@@ -245,6 +251,7 @@ const CreateLinkForm: FunctionComponent<CreateLinkFormProps> = ({
                 buttonText="Browse"
                 fileSizeText={file ? formatBytes(file.size) : ''}
```
[the remark this answers is not in the document]
Comment (gweiying) [reply src/client/user/components/CreateUrlModal/CreateLinkForm.tsx:252]: thanks for reminding me! didn't see it, I'll standardise them.

[the remark this answers is not in the document]
Comment (gweiying) [reply src/client/user/components/CreateUrlModal/CreateLinkForm.tsx:251]: fixed here! [f293a7e](https://github.com/opengovsg/GoGovSG/pull/2169/commits/f293a7e7c897a03e366ad7bbe1bc87df22a4571a)

**Review by halfwhole — APPROVED** [review]
lgtm, thanks!

**Pull request merged**